dist/plugin.json

```diff
@@ -6,8 +6,8 @@
   "info": {
     "description": "Pie chart panel for grafana",
     "author": {
-      "name": "Raintank Inc.",
-      "url": "http://raintank.io"
+      "name": "Naudit HPCN S.L.",
```

Comment:
@gretamosa The only information about a contribution is the git info. ex name and email. You could recreate the PR using other credentials if you like. 

But the author field in the plugin.json should not change because of one pull request.